taxi_service/settings.py

```diff
@@ -56,7 +56,9 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [
+            os.path.join(BASE_DIR, "taxi/../templates")
```

Comment:
"taxi/../templates" == "templates" you don't need to enter and exit the folder.

Reply:
Yep, my fault, thanks)